README.md

```diff
@@ -240,4 +240,12 @@ As of February 2021, index resources that don't support tags aren't applicable t
 May 2020: Microsoft.DocumentDB/databaseAccounts/ipRangeFilter updated from a string property to an array.  Please re-author your custom definitions to support the property as an array.  
 July 2020: The alias Microsoft.Sql/servers/securityAlertPolicies/emailAddresses[] and related policies were deprecated. 
 
+### AKS Policy does not current support Application gateway Ingress controller and Nginx controller currently 
```

Comment:
```suggestion
### Azure Policy for AKS currently does not support Application Gateway ingress controllers or Nginx controllers
```

Reply:
@nehakulkarni123 - Ingress issue is fixed and deployed to Public, Fairfax, Mooncake clouds now.

Reply:
excellent! will remove